packages/@o3r/test-helpers/src/utilities/package-manager.ts

```diff
@@ -40,6 +42,7 @@ const PACKAGE_MANAGERS_CMD: {[packageManager in SupportedPackageManagers]: {[com
   },
   yarn: {
     add: ['yarn', 'add'],
+    ci: ['yarn', 'install', '--frozen-lockfile'],
```

Comment:
Stupid question: `--frozen-lockfile` is only for yarn 1, is it the one we are using here?

Reply:
should be `--immutable` I guess, my bad